### PR TITLE
Page Builder migration docs release

### PR DESCRIPTION
**Pull request by bdenham**
## Purpose of this PR

This PR will release the Page Builder migration docs to production (accessible within the existing Page Builder docs).

## Special Instrtuctions

The pipelines need to be updated and tested before publishing these docs to production.

whatsnew
Page Builder migration documentation - Learn how to use the Page Builder migration module to [migrate your BlueFoot content to Page Builder](https://devdocs.magento.com/page-builder-migration/docs/).

### CODE REVIEW

File: _data/toc/page-builder.yml [hunk 1]
```diff
@@ -141,6 +141,34 @@ pages:
     url: /page-builder/docs/how-to/how-to-deactivate-pagebuilder.html
     versionless: true
 
+- label: BlueFoot content migration
+  versionless: true
+  children:
+
+  - label: Overview of content migration
+    url: /page-builder-migration/docs/
```
Comment (dshevtsov) [inline _data/toc/page-builder.yml]: should it be `/page-builder-migration/docs/index.html`?

Comment (bdenham) [reply _data/toc/page-builder.yml]: @belbiy recommended without the file name.

File: _data/toc/page-builder.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ label: Page Builder
 versionless: true
 pages:
 - label: What is Page Builder?
-  url: /page-builder/docs/
+  url: /page-builder/docs/index.html
```
Comment (jeff-matthews) [inline _data/toc/page-builder.yml]: Should this be extensionless, like line 149?

File: _data/toc/page-builder.yml [hunk 3]
```diff
@@ -141,6 +141,34 @@ pages:
...
+- label: BlueFoot content migration
+  versionless: true
+  children:
+
+  - label: Overview of content migration
+    url: /page-builder-migration/docs/
+    versionless: true
+
+  - label: Install the migration module
+    url: /page-builder-migration/docs/install-migration-module.html
```
Comment (jeff-matthews) [inline _data/toc/page-builder.yml]: Should all of these links also be extensionless like line 149?

File: _data/toc/page-builder.yml (hunk 2 above)
Comment (bdenham) [reply _data/toc/page-builder.yml]: It's fine as is but I'll make it consistent.

File: _data/toc/page-builder.yml (hunk 3 above)
Comment (bdenham) [reply _data/toc/page-builder.yml]: No, just the index.html, unless they can be extensionless?

File: Docfile.yml [hunk 4]
```diff
@@ -24,3 +24,8 @@ content_map:
   repository: magento-devdocs/magento2-page-builder
   branch: develop
   filter: true
+-
+  directory: page-builder-migration
+  repository: magento-devdocs/magento2-page-builder-data-migration
+  branch: 1.0.0-release
+  filter: true
```
Comment (dshevtsov) [inline Docfile.yml]:
```suggestion
  filter: true
  
```
The file should end with a blank line

**Comment by bdenham** [comment]
Has this been done?
> Add the pbm task to prerequisites of the subrepos task.

**Comment by dshevtsov** [comment]
@bdenham please use only one label to highlight type of the changes: Major update or New topic

**Comment by dshevtsov** [comment]
> Has this been done?
No. In the `rakelib/update.rake` file, you should add the new task added in this pull request changing the subrepos task from:
```
task subrepos: %w[m1 mbi pb mftf v2_0]
```
to
```
task subrepos: %w[m1 mbi pb pbm mftf v2_0]
```

**Comment by bdenham** [comment]
Done. Thanks.

File: Docfile.yml [hunk 5]
```diff
@@ -24,3 +24,9 @@ content_map:
   repository: magento-devdocs/magento2-page-builder
   branch: develop
   filter: true
+-
+  directory: page-builder-migration
+  repository: magento-devdocs/magento2-page-builder-data-migration
```
Comment (dshevtsov) [inline Docfile.yml]: Should it be changed to `magento/magento2-page-builder-data-migration`?

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**